src/types/response/transaction-api/TransactionList.type.ts

```diff
@@ -0,0 +1,28 @@
+export type Transaction = Omit<
```

Comment:
This type is the new created after fetched from the API : we receive a `TransactionDTO` object and we rebuild a `Transaction` to make it easier tu use